admin-base/ui.apps/src/main/content/jcr_root/apps/field/material-range/template.vue

```diff
@@ -1,25 +1,25 @@
 <template>
-  <div v-if="!schema.preview" class="range-field" :class="{'is-empty': !value}">
+  <div v-if="!schema.preview" class="range-field" :class="{'is-empty': isBlank}">
     <button class="range-btn" @click="onRangeBtnClick">
-      <admin-components-icon :icon="icon" :lib="IconLib.MATERIAL_ICONS"/>
-      <span v-if="!value" class="strike"></span>
+      <span v-if="isBlank" class="strike"></span>
+      <admin-components-icon icon="linear_scale" :lib="IconLib.MATERIAL_ICONS"/>
```

Comment:
there a reasoin you swapped it?

Reply:
Noce catch, they must have changed while committing lines instead of hunks inside Atlassian SourceTree somehow. I'll reverse the change.